sass/themes/cr/2017/variables/_colours.scss

```diff
@@ -24,3 +24,9 @@ $colour-dark-purple-light: #400344;
 $colour-royal-blue-light: #584bfe;
 $colour-mint-green-light: #8df7dd;
 $colour-red-light: #ff044c;
+
+
+/* standard form colors */
+$colour-scorpion-grey:#595959;
+$colour-ribbon-red: #EA0043;
+$colour-persian-purple: #340F78;
```

Comment:
no need to do this as the colour variables from the base theme are already imported in the cr sub theme.
If you want to see the new form styling here you'd need to import the standard-form.scss from the base theme rather than the form.scss